backend/src/user/user.controller.ts

```diff
@@ -0,0 +1,14 @@
+import {UserService} from "./user.service";
+import {Body, Controller, Post} from "@nestjs/common";
+import {Prisma} from "@prisma/client";
+
+@Controller('users')
+export class UserController {
+    constructor(private readonly userService: UserService) {
+    }
+    @Post('create-user')
+    createNewUser(@Body() newUserData:  Prisma.UserCreateInput) {
+        this.userService.createUser(newUserData)
+        return "Pomyślnie dodano użytkownika o adresie email: "+ newUserData.email + " oraz nazwie: "+newUserData.login
```

Comment:
zwróć jednak uwagę że najpierw zostanie zwrócona wiadomość o pomyślnym dodaniu, a dopiero później zostanie to faktycznie dodane. Spróbuj wysłać nieprawidłowe dane dla serwisu, ale takie które są poprawne dla kontrolera